### DISCUSSION TITLE
Confused on ORP operation.

### DISCUSSION THREAD

**Discussion by jamestalmage** [tagyoureit/nodejs-poolController#857] [Nixie]
I've got a Nixie REM/Chem build using the Sequent MegaBas (and relay, etc) and Atlas Scientific ORP/ph. It follows the Wiki pretty darn close. I chlorinate with the Intellichlor IC40.

I'm a bit puzzled on the ORP auto-dosing behavior. It seems to only chlorinate 10-15 minutes at a time, then is in an "ORP Mixing" state for 5-ish minutes before dosing again.

This leaves the chlorinator sitting idle about 30% of the time the pump is running.

I would have assumed the mode of operation would have just been "Come on at 100% until you see the target ORP reached".

What's the purpose of this mode of operation (if it is indeed how it's supposed to work)?

**Comment by tagyoureit** [comment]
The chlorine takes time to circulate and mix in the pool.  If we dose at 100% until the target is reached, we would overshoot the target.  By allowing the mix time, it gives the actual concentration of chlorine in the pool to be dispersed and reach the ORP probe.

**Comment by jamestalmage** [comment]
Hmmm. Is it really that pronounced of a problem? In a 10,000 gallon pool, I need to run like 3 hours with an IC40 just to add 2ppm.

How long is long enough for a pool my size to properly mix? Isn't it like, at least, 30 minutes? Even if I over-shoot, it's by 0.5ppm.

Maybe scale it based on how far from the target you are? (More frequent stops for mixing the closer to the target you get?)

**Reply by tagyoureit** [reply]
Our goal with the dosing is to aim for consistency with the chemical readings.
